src/gcp_scanner/crawl.py

```diff
@@ -125,9 +124,7 @@ def get_compute_instances_names(
     while request is not None:
       response = request.execute()
       if response.get("items", None) is not None:
-        for _, instances_scoped_list in response["items"].items():
-          for instance in instances_scoped_list.get("instances", []):
-            images_result.append(instance)
+        images_result = [instance for _, instances_scoped_list in response["items"].items() for instance in instances_scoped_list.get("instances",[])]
```

Comment:
Split to multiple lines for readability.

Reply:
Hi @ZetaTwo , I have resolved this issue in the latest PR.